src/styles/globalStyle.ts

```diff
@@ -0,0 +1,58 @@
+import { createGlobalStyle, css } from 'styled-components';
+
+import { type RiskLevels } from '@/constants/abacus';
+
+export const GlobalStyle = createGlobalStyle`
+  :root {
+    /* Computed: Layers */
```

Comment:
no longer a computed layer. comment should be removed

Reply:
I don't think any of these layers are `computed` anymore. The `computed` part was based off of using a dynamic hue, lightness, saturation to derive other layer colors.